.idea/codeStyleSettings.xml

```diff
@@ -493,4 +493,4 @@
     </option>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
   </component>
-</project>
+</project>
```

Comment:
Intellij obsessively removes the extra line at the end of this file so whatever.